### DISCUSSION TITLE
Suggestion: Add option to limit number of depots

### DISCUSSION THREAD

**Discussion by greeny** [voxeltycoon/issues#853] [General]
On behalf of Nevesbr on Discord, I'm suggesting an option to limit number of depots to 1 per transport type (so 1 road depot, 1 rail depot).

- would be enabled on start game screen
- would be saved in a game
- would limit player to build one depot per vehicle type

I'm not sure how to handle multiple tiers of depots. Either allow user to delete depot to build new (better) one (but that can be abused to "move" the depot everytime it's needed) or use it together with option to upgrade depot in place.

**Comment by victorleblais** [comment]
What is the purpose of this?

Couldn't you just "force" yourself to only place one?

**Comment by ondrach2009** [comment]
good
